http/http.go

```diff
@@ -12,8 +12,8 @@ import (
 
 	"encoding/hex"
 
-	"github.com/nwidger/nintengo/m65go2"
-	"github.com/nwidger/nintengo/nes"
+	"github.com/kaicheng/nintengo/m65go2"
+	"github.com/kaicheng/nintengo/nes"
```

Comment:
I am Sorry...

I used "go get" to get my repo cloned from nwidger's. This is only an temp developing version, and I mistakenly clicked the "Creat Pull Request". I must clarify here that I am not going to "steal" or "claim" any work of others. I definately will change back all the names after I finished all the cleanup stuffs.